### ISSUE TITLE
Dashboard - Add ability to see logs per available loggers

### ISSUE THREAD

**Issue by efonsecab** [dotnet/aspire#1313]
I have an app which was a project with 2 Background Services.
Currently, the dashboard show the logs messages in the same window.
It would be useful to have the ability to select the Logger, to filter for only message on specific loggers.

Screenshot 1
<img width="935" alt="image" src="https://github.com/dotnet/aspire/assets/3481899/5a9dac9c-9204-4007-9fa7-25f877024dde">

Screenshot 2
<img width="228" alt="image" src="https://github.com/dotnet/aspire/assets/3481899/2f9f2521-9e1e-4f7a-99c8-f4739be02d05">

Screenshot 3
<img width="560" alt="image" src="https://github.com/dotnet/aspire/assets/3481899/9a25392b-4bb5-461c-86f3-2ee230ec34d6">

Screenshot 4
<img width="583" alt="image" src="https://github.com/dotnet/aspire/assets/3481899/c9f7246d-90ce-4b6e-a288-201338518455">

**Comment by mitchdenny** [comment]
This would be pretty awesome.

**Comment by davidfowl** [comment]
This should be a structured logs feature not a console logs feature (it was also added in preview 2 afaik).

cc @JamesNK

**Comment by mitchdenny** [comment]
Quite right ... it is there: 
![image](https://github.com/dotnet/aspire/assets/513398/de015746-86dd-4b77-8221-75eec9d5282d)

**Comment by JamesNK** [comment]
The logger category is an option in the structured logs filter dialog. That was added a few days ago. I think it will be available in preview 3?

I considered making it a dropdown on the main page, but there is limited UI space, and a logger name can be very long (name space + type name).

See https://github.com/dotnet/aspire/pull/1214

**Comment by mitchdenny** [comment]
Yeah, I think what you have in main is good for now. I think we can close this as done.

**Comment by efonsecab** [comment]
Useful, yet sadly not enough.
The benefit with the logs section is that it's a console instead of a grid.
As for space, I think menus could eventually be converted into panes, similar to what Azure Portal does, but that one would be another task.

**Comment by JamesNK** [comment]
The "console logs" page is displaying what is written to the console. It's all text as far as the dashboard is concerned.

If you want a more structured experience (filtering, log levels, metadata, etc) then you need to use structured logs.
